src/06-hello-world/README.md

```diff
@@ -104,7 +104,7 @@ Breakpoint 1, main () at src/06-hello-world/src/main.rs:10
 10          let mut itm = aux6::init();
 ```
 
-Note that there's a `.gdbinit` at the root of the Cargo project. It's pretty similar to the one we
+Note that there's a `openocd.gdb` at the root of the Cargo project. It's pretty similar to the one we
```

Comment:
It looks like `openocd.gdb` used to be named `.gdbinit`, and this doc wasn't updated when the file name changed.